cypress/integration/plugins/security/audit_log_spec.js

```diff
@@ -42,16 +42,21 @@ if(Cypress.env("SECURITY_ENABLED")) {
         () => {
           cy.get('button[name="auditLoggingEnabledSwitch"]').first().click({ force: true })
         }
-      ).then((response) => {
-          const body = JSON.parse(response.response.body);
-  
-          expect(body.message).to.equal("'config' updated.");
+      ).then((result) => {
+          // NOTE: JSON.parse fails on ARM64 because it is an object
+          try {
+            const body = JSON.parse(result.response.body);
+            expect(body.message).to.equal("'config' updated.");
+          } catch (e) {
```

Comment:
could we specify the concrete exception type in catch block? instead of a general exception

Reply:
MDN docs suggest I can catch (e)

and then do 

`if (e instanceof SyntaxError)` if you would like?

Reply:
One sec while I verify

Reply:
awesome, thanks